README.md

```diff
@@ -242,36 +242,38 @@ In cluster mode user may have to increase the concurrency value in order to achi
 In cluster mode you can control the number CPU's by changing the configuration file in the cluster mode. If not set, it takes the default number of CPU's (number of logical threads) available.
 
 Running application server in microservices mode (default port: 3000):
-  - run “npm install” to install all dependencies for each service directory (don't use sudo), and start each service separately
-  - To auto run npm install run the script 
-    - python npm_install_all.py
-    - Start each service in separate terminal window.
+- run `npm install` to install all dependencies for each service directory (don't use sudo)
```

Comment:
Please avoid the interior bullet. Please add the sentence "To auto run npm install in each directory run the script python npm_install_all.py" in one paragraph that begin with "run npm install to install all dependencies for each service directory (don't use sudo)". Please emphasize that "npm install" is a proper command.